src/util/statistics_public.cpp

```diff
@@ -30,7 +30,6 @@ void registerPublicStatistics(StatisticsRegistry& reg)
   reg.registerHistogram<api::Kind>("api::TERM", false);
 
   reg.registerValue<std::string>("driver::filename", false);
-  reg.registerValue<std::string>("driver::sat/unsat", false);
```

Comment:
This change looks unrelated.

Reply:
Discussed offline. The change is on purpose and this is fine with me.

Reply:
It technically is, but fits well with this PR